var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -238,6 +238,11 @@ class Kokkos(CMakePackage, CudaPackage, ROCmPackage):
     # Patches
     patch("hpx_profiling_fences.patch", when="@3.5.00 +hpx")
     patch("sycl_bhalft_test.patch", when="@4.2.00 +sycl")
+    patch(
```

Comment:
Can you add a brief comment above the `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed

?

Reply:
Will do. I'm afk until early January due to travel. I'll push it as soon as I can.

Reply:
@alalazo added the comment.